src/editors/collectionDocumentsProvider.ts

```diff
@@ -60,12 +60,15 @@ export default class CollectionViewProvider
     // Ensure we're still connected to the correct connection.
     if (connectionId !== this._connectionController.getActiveConnectionId()) {
       operation.isCurrentlyFetchingMoreDocuments = false;
+      const oldConnectionName =
+        this._connectionController.getSavedConnectionName(connectionId || '') ||
+        'the connection';
```

Comment:
“no longer connected to the connection” sounds a bit odd, right? Maybe this could say “no longer connected to the database”?

Reply:
That reads much better yup. Updated